src/services/productService.js

```diff
@@ -0,0 +1,27 @@
+const productDao = require('../models/productDao');
+
+const getProducts = async(productId,userInfo,productInfo)=>{
+    // 토큰 값 꺼내 저장
+    const userId = userInfo.id;
+    const userEmail = userInfo.email;
+
+    // DB에 토큰에 꺼낸 값들을 넣어 비교 
+    const checkUser = await productDao.getUsers(userId,userEmail);
+    // DB에 꺼낸 id와 email을 변수에 따로 저장 (이유 : 해당 사항이 없다면 조건문에 걸려 return 하기 위해)
+    const dbUserId = checkUser[0].id;
+    const dbUserEmail = checkUser[0].email;
+
+    const products = await productDao.getProducts(productId);
+    if(userId !== dbUserId || userEmail !== dbUserEmail) {
+        return "해당 유저가 없습니다.";
+    }
+    return products;
+};
+
```

Comment:
```suggestion

    if(userId !== dbUserId || userEmail !== dbUserEmail) {
        return "해당 유저가 없습니다.";
    }
    
    const products = await productDao.getProducts(productId);
    return products;
};
```

- productDao에 있는 getProducts를 하기 전에 이용자를 먼저 검증하는 것이 절차적으로 순서일 것 같은데 맞을까요?